src/OowpQuery.php

```diff
@@ -40,32 +40,32 @@ function __construct($query = '')
 
     /* Interfaces */
 
-    public function getIterator()
+    public function getIterator(): Traversable
```

Comment:
need to import this first

Reply:
Did you get an error? I thought it was a integral part of PHP